spec/services/preference_set_table_state_service_spec.rb

```diff
@@ -28,7 +28,7 @@
       it 'raises an error' do
         expect {
           subject.update!(table_state: table_state2, table_name: table_name)
-        }.to raise_error(PreferenceSetTableStateService::UpdateFailedError, "Failed to update table state for table_name: '#{table_name}'")
```

Comment:
can't comment above, but we should be capturing the `#save!` and raising an ActiveRecord error.